src/cmake/openvino.cmake

```diff
@@ -41,29 +41,17 @@ target_include_directories(${TARGET_NAME} PUBLIC
 
 find_package(MLIR REQUIRED CONFIG)
 
-set(MLIR_OPENVINO_LIBS
-        MLIRAnalysis
-        MLIRExecutionEngine
-        MLIRIR
-        MLIRJitRunner
-        MLIRLLVMDialect
-        MLIRLLVMToLLVMIRTranslation
-        MLIRToLLVMIRTranslationRegistration
-        MLIRParser
-        MLIRTargetLLVMIRExport
-        MLIRSupport
-        MLIROptLib
-        LLVMX86AsmParser
-        MLIRFuncDialect
-        MLIRFuncAllExtensions
-        MLIRUBToLLVM)
+get_property(MLIR_ALL_LIBS GLOBAL PROPERTY MLIR_ALL_LIBS)
 
 target_link_libraries(${TARGET_NAME} PRIVATE openvino::reference
                                              openvino::shape_inference
                                              openvino::pugixml
                                              ${CMAKE_DL_LIBS}
                                              Threads::Threads
-                                             ${MLIR_OPENVINO_LIBS})
+                                             ${MLIR_ALL_LIBS})
+
+add_tpp_mlir_libs(${TARGET_NAME})
+
```

Comment:
```suggestion
add_tpp_mlir_includes(${TARGET_NAME})

```

Reply:
Shouldn't affect the build, as no code outside `transformations` use these headers. And the build of `transformations` uses this directive.